Packs/Lokpath_Keylight/.pack-ignore

```diff
@@ -1,2 +1,5 @@
 [file:Lockpath_KeyLight_v2.yml]
 ignore=IN126,BA108,BA109
+
+[file:README.md]
+ignore=RM105
```

Comment:
why are we ignoring it here?

Reply:
it's for examples, which I did not change